src/NServiceBus.Core/Performance/Counters/PerformanceCounterHelper.cs

```diff
@@ -33,7 +33,7 @@ static bool TryToInstantiatePerformanceCounter(string counterName, string instan
                 throw new Exception($"The endpoint name ('{instanceName}') is too long (longer then {(int) sbyte.MaxValue}) to register as a performance counter instance name. Reduce the endpoint name.");
             }
 
-            var message = $"NServiceBus performance counter for '{counterName}' is not set up correctly. To rectify this problem download the latest powershell commandlets from http://www.particular.net downloads page.";
+            var message = $"NServiceBus performance counter for '{counterName}' is not set up correctly. To rectify this problem download consult the documentation regarding performance counters at https://docs.particular.net.";
```

Comment:
> download consult the documentation

sounds wrong ;)

This seems like the first time we really point to the docs url. I only know statements like "consult the documentation". Should we be consistent with that?